### ISSUE TITLE
Accessibility audit: (JAWS) Get in touch channel # announcement

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#312]
## Describe the issue
JAWS reads out # as number not ‘hashtag’ in the #design-system channel copy.

## Steps to reproduce the issue
With Voiceover on, tab to the link to listen to the announcement of 'number' not #. 

## Actual vs expected behaviour
'#' is announced.

## Fix recommendations

- [x] Wrap the # in a span with an aria-label="hashtag"

## Related stories

- [Accessibility audit: (JAWS) Slack channel link not public](https://github.com/dwp/design-system/issues/311)
